pcp/Platform.c

```diff
@@ -502,6 +506,8 @@ void Platform_init(void) {
 
 void Platform_done(void) {
    pmDestroyContext(pcp->context);
+   pmFreeResult(pcp->result);
```

Comment:
Shouldn't the result be freed before freeing the context? Which resource is dependent on which?

Reply:
There's no dependency there - doesn't matter which happens first.